app/src/main/java/com/bignerdranch/android/criminalintent/models/CrimeListViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.bignerdranch.android.criminalintent.models
+
+import androidx.lifecycle.ViewModel
+
+class CrimeListViewModel : ViewModel() {
+  companion object {
+    const val LIST_LENGHT = 100
+  }
+
+  val crimes: MutableList<Crime> = mutableListOf()
+
+  init {
+
+    for (i in 0 until Companion.LIST_LENGHT) {
```

Comment:
remove the `Companion.`

Reply:
also maybe a method (because it's mocked data)